tests/active_features/bootstrap/FeatureContext.php

```diff
@@ -30,8 +30,8 @@ public function __construct($parameters)
         $this->cliroot          = dirname(dirname(__DIR__)) . '/..';
         $this->parameters      = $parameters;
         $this->start_time      = time();
-        $this->cache_file_name = $_SERVER['HOME'] . '/.terminus/cache/session';
-        $this->cache_token_dir = $_SERVER['HOME'] . '/.terminus/cache/tokens';
+        $this->cache_file_name = $_SERVER['HOME'] . '/.terminus/testcache/session';
+        $this->cache_token_dir = $_SERVER['HOME'] . '/.terminus/testcache/tokens';
         $this->connection_info = ['host' => $parameters['host'], 'machine_token' => $parameters['machine_token'],];
```

Comment:
Generally I'd prefer something determined from the ENV {dev|test|ci|prod(user in this case?)} but am 2/10 on getting that and 6/10 on getting this just to work.